inc/compatibility/elementor.php

```diff
@@ -456,6 +456,24 @@ public static function is_elementor_template( $elementor_template_type ) {
 		return self::$cache_cp_has_template[ $location ];
 	}
 
+	/**
+	 * Detect if the a page is using the checkout widget.
+	 */
+	public static function is_elementor_checkout() {
```

Comment:
We can leverage the class static property to cache the result of that method (::is_elementor_checkout) in terms of performance.

Reply:
I think currently that method is used by two where in Neve repo (by front_end.php and woocommerce.php). Also, we'll leverage that method from Neve Pro. So, at least 3 call for now.